examples/rckks/main.go

```diff
@@ -0,0 +1,770 @@
+package main
+
+import (
+	"fmt"
+	"github.com/ldsec/lattigo/v2/ckks"
+	"github.com/ldsec/lattigo/v2/ring"
+	"github.com/ldsec/lattigo/v2/rlwe"
+	"github.com/ldsec/lattigo/v2/utils"
+	"runtime"
+	"time"
+)
+
+// LogN of the ring degree of the used parameters
+var LogN = 15
+
+// Benchmarking between emulated complex using RCKKS and CKKS
```

Comment:
I am still not convinced this is really a good example to integrate. It is a very long program and is currently insufficiently described.

Reply:
Move to a Gist.